doc/library-user.rst

```diff
@@ -72,6 +72,12 @@ handled as follows:
 2. If the fixed issue is non-critical, it will be delayed until the next major
    release (following the normal Neptune guarantees).
 
+Also note that a breaking patch release will be merged into subsequent patch and
+minor releases, which *will not* be explicitly marked as containing breaking
```

Comment:
Didn't we say that it should be added to subsequent patch and minors too?

Reply:
There was no consensus on it.

Reply:
Should we reach consensus before updating the docs or do you prefer to update them first and then discuss + update again?

Reply:
It's probably better to document the current / previous procedure first, I think.